app/modules/engagement/views/events.py

```diff
@@ -17,6 +17,7 @@ def dashboard(request):
     context = {
         'user'   : request.user,
         'events' : user_events,
+        'user_has_event' : request.user in [x.user for x in user_events]
```

Comment:
No need to do this, event can reverse access their event set using the related names in the models set in engagement module... I think I set it to like `events_registered` so it'd be `user.events_registered`